core/services/ardupilot_manager/mavlink_proxy/MAVLinkRouter.py

```diff
@@ -64,9 +64,9 @@ def convert_endpoint(endpoint: Endpoint) -> str:
         if master_endpoint.connection_type == EndpointType.TCPServer:
             # The "--tcp-port 0" argument is used to prevent the router from binding TCP port 5760
             return f"{self.binary()} \
```

Comment:
The code is correct but we can add the `5777` port over the removed "Internal Link".

Reply:
Done